tests/cisco_asa/show_running-config_object_network/cisco_asa_show_running-config_object_network.raw

```diff
@@ -2,15 +2,13 @@ object network corp
  subnet 10.0.0.0 255.0.0.0
 object network internal
  subnet 192.168.0.0 255.255.255.0
-object network any
```

Comment:
Why is this being adjusted?

Reply:
ASA won't allow the creation of objects with the name 'any'  so, I had to remove it from my test. 

```
asa_test(config)# object network any
ERROR: This object name is reserved
```